Client/Blogger/src/Redux/Feature/BlogSlice.js

```diff
@@ -0,0 +1,42 @@
+import { createSlice } from "@reduxjs/toolkit";
+
+export const BlogSlice = createSlice({
+  name: "blog",
+  initialState: {
+    blogList: [],
+    isMediaSelected: false,
+    searchList: [],
+    location: "none",
+    searchValue: "",
+  },
+  reducers: {
+    fetchBlog: (state, action) => {
+      console.log("fetch blog called");
```

Comment:
During the development, it's common to use console logs for debugging purposes. But, it's important to remove unnecessary console logs before initiating a code review, unless they are essential for the functionality or serve a purpose